pkg/record/schema/proto/mutable_test.go

```diff
@@ -114,7 +114,7 @@ func TestMutableSchema_SetDescriptors_Panics(t *testing.T) {
 
 			ms := fileDescriptorSetToMutalbeSchema(t, getFileDescriptorSet(t, standaloneDescriptorSetPath))
 			ms.SetDescriptors(tc.descriptors)
-			is.True(false)
+			is.True(false) //expected panic
```

Comment:
IMO it would be clearer with `is.Fail() // expected panic`.

Reply:
Agreed. :+1: